.buildkite/pipeline.yml

```diff
@@ -198,23 +198,6 @@ steps:
         retry:
           manual:
             allowed: true
-      - label: "K8s integration tests"
-        key: "k8s-integration-tests"
-        env:
-          K8S_VERSION: "v1.30.2,v1.29.4,v1.28.9"
```

Comment:
Noticed that you changed this to only v1.30.2, does that mean we are not testing against the other stable versions of kubernetes? If so, is that desired? Thanks.

Reply:
so this change means that `mage integration:kubernetes` indeed will run the test against a cluster that its version is defined in the env var of  K8S_VERSION. However this PR added also `mage integration:kubernetesMatrix`, which is what is invoked in our CI, that runs the k8s integration tests against multiple k8s versions defined [here](https://github.com/elastic/elastic-agent/blob/main/pkg/testing/kubernetes/supported.go#L22)

Reply:
Ha, it makes sense now. Thanks for the explanation!